README.md

```diff
@@ -38,6 +38,10 @@ To quickly test if one can build with go-ceph on your system, run:
 go get github.com/ceph/go-ceph
 ```
 
+Once compiled, code using go-ceph is expected to dynamically link to the Ceph
+libraries. These libraries must be available on the system where the go based
+binaries will be run. The use of cgo does not allow for fully static binaries.
```

Comment:
I think this is not completely correct. It is probably possible to link C libraries statically into the GO executable. However, Ceph (packages) does not provide the option to statically link the libraries (no `.a` files, only `.so` ones).

Reply:
I will change it to "Our use of cgo and ceph libraries" instead of "The use". I think that makes it more accurate.

Reply:
Updated. If you have issues with the new version please give a shout. Otherwise, I'll merge later this week.